gap_analysis_crm/README.rst

```diff
@@ -0,0 +1,79 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+{gap_analysis_crm}
+==============
+
+linking business requirements to opportunity.
```

Comment:
This module is part of a set ("Gap analysis") and extend the basic functionality
to link the business analysis to CRM opportunity.